README.md

```diff
@@ -203,8 +203,8 @@ OpenFunction team has presented OpenFunction project in some community events in
 
 OpenFunction is sponsored and open-sourced by the [KubeSphere](http://kubesphere.io/) Team and maintained by the OpenFunction community.
 
-- Slack: [#sig-serverless](https://kubesphere.slack.com/archives/C021XAR3CG3)
-- Wechat: join the OpenFunction user group by following the KubeSphere WeChat subscription
+- Slack: [#openfunction](https://slack.cncf.io/)
```

Comment:
is it better to use "https://cloud-native.slack.com/archives/C03ETDMD3LZ"?

Reply:
Added as another option